dev/node_app/src/app.ts

```diff
@@ -73,7 +73,7 @@ const channelHeaderBindings = {
             location: 'send-button',
             icon: 'icon.png',
             label: 'send hello message',
-            call: {
+            submit: {
                 path: '/send',
```

Comment:
```suggestion
                path: '/send/submit',
```

Reply:
Can you help me understand how this one would work? The channel header button needs to ask the user for input because they're merely pressing a button instead of passing in parameterized arguments via a slash command. If the button sends them right to `send/submit` then we're skipping the form collection stage. But if we put the form in the `/send/submit` path, then it doesn't make sense for the slash command because they've already passed in the parameter they want, no? Is there an idea way to cater to both that you recommend? Is there some kind of arg checking we could do to determine if they've already passed in parameter(s) via a slash command?